rcl/include/rcl/timer.h

```diff
@@ -225,7 +225,8 @@ rcl_timer_fini(rcl_timer_t * timer);
  * \param[inout] timer the handle to the timer to call
  * \return #RCL_RET_OK if the timer was called successfully, or
  * \return #RCL_RET_INVALID_ARGUMENT if any arguments are invalid, or
- * \return #RCL_RET_TIMER_INVALID if the timer is invalid, or
+ * \return #RCL_RET_INVALID_ARGUMENT if the timer is invalid, or
+ * \return #RCL_RET_TIMER_INVALID if the timer->impl is invalid, or
```

Comment:
I do not think we need to change here, so other places. line 227 already describes `RCL_RET_INVALID_ARGUMENT`.